src/TransactionFormatterTest.java

```diff
@@ -0,0 +1,56 @@
+import org.junit.Test;
+
+import java.time.*;
+import java.util.Date;
+
+import static org.junit.Assert.*;
+
+public class TransactionFormatterTest {
+
+    private TransactionFormatter target = new TransactionFormatter();
+
+    @Test
+    public void testTransactionFormatterStringForDebit() {
+
+        TransactionType type = TransactionType.DEBIT;
+        String description = "Test";
+        long money = -400;
+        Date date = Date.from(LocalDate.of(2000, Month.JANUARY, 1)
+                                        .atTime(LocalTime.NOON)
+                                        .toInstant(ZoneOffset.UTC));
+
+        final Transaction transaction = new Transaction();
+        transaction.setType(type);
+        transaction.setDescription(description);
+        transaction.setMoney(money);
+        transaction.setDate(date);
+
+        String transactionAsAString = target.format(transaction);
+        assertNotNull(transactionAsAString);
+        assertNotEquals("",transactionAsAString);
+        assertEquals("DEBIT, Test, £4.00, 2000-01-01", transactionAsAString);
```

Comment:
This line would fail if `transactionAsAString` is `null` or `""` anyway, so we can get rid of the two assertions above